frontend/server/src/Controllers/Badge.php

```diff
@@ -145,6 +146,25 @@ private static function getBadgeDetails(string $badgeAlias): array {
             'owners_count' => $ownersCount,
         ];
     }
+    /**
+     * @return array{smartyProperties: array{payload: BadgeListPayload, title: string}, entrypoint: string}
+     */
+    public static function getBadgeListForSmarty(\OmegaUp\Request $r) {
+        $r->ensureIdentity();
+        $badges = self::apiList($r);
+        $owned_badges = self::apiMyList($r);
+        print_r($owned_badges);
+        return [
+            'smartyProperties' => [
+                'payload' => [
+                    'badges' => $badges,
+                    'owned_badges' => $owned_badges['badges']
```

Comment:
nit:

```suggestion
                    'ownedBadges' => $ownedBadges['badges']
```

en los APIs se usa snake_case, pero este no es un API, así que sí se vale usar camelCase.